src/uvw/util.hpp

```diff
@@ -11,6 +11,12 @@
 #include <array>
 #include <uv.h>
 
+#ifdef _WIN32
```

Comment:
Mine is a Window 10 Home x64 machine with Visual studio Community 2015 Update 3.
It is very close (except for the OS flavor of course) to the image used by AppVeyor.
Here are the test results from AppVeyor for [Win32](https://ci.appveyor.com/project/tusharpm/uvw/build/1.0.17/job/4khwdohx7372h5da/messages) and [x64](https://ci.appveyor.com/project/tusharpm/uvw/build/1.0.17/job/6f45r2rfh0lk52h9/messages) too, just for closure.